pkg/mosn/mosn.go

```diff
@@ -126,6 +122,32 @@ func (m *Mosn) inheritHandler() error {
 	return nil
 }
 
+// replace your own inherit func with default inherit func
+func InitInheritFunc(f func (c *v2.MOSNConfig) error) {
```

Comment:
Where is this func called?

Reply:
> Where is this func called?

If we wants to customize  inheritFunc, we can call this function in main.go to replace the default inheritFunc